CONFIGURATION.md

```diff
@@ -1,79 +1,171 @@
-# Configuration
+# SolarWinds APM Ruby Configuration Guide
 
-By default all applicable instrumentations are enabled. The only required configuration is the service key, so a minimal example to get started is:
+This guide covers all configuration options for the SolarWinds APM Ruby gem, an OpenTelemetry-based distribution that provides automatic instrumentation and observability features for Ruby applications.
+
+## Table of Contents
+
+- [Quick Start](#quick-start)
+- [Configuration Precedence](#configuration-precedence)
+- [Environment Variables](#environment-variables)
+- [Configuration Files](#configuration-files)
+- [Programmatic Configuration](#programmatic-configuration)
+- [Advanced Configuration](#advanced-configuration)
+- [Configuration Reference](#configuration-reference)
+- [Troubleshooting](#troubleshooting)
```

Comment:
I'd argue that a manually maintained TOC is no longer necessary--as long as each section has the proper header formatting, we can rely on the automatic Outline view for GH markdowns:

<img width="1412" height="428" alt="Image" src="https://github.com/user-attachments/assets/d10bbe95-f063-4ed9-bf24-eee8a1224826" />

Reply:
Removed.